workspace_host/interface.js

```diff
@@ -693,6 +693,7 @@ function _recursiveDownloadJobManager(curDirPath, S3curDirPath, callback) {
 }
 
 async function _syncInitialZipAsync(workspace) {
```

Comment:
```suggestion
async function _getInitialZipAsync(workspace) {
```

I think using "sync" to mean "get"/"fetch" is confusing.